### DISCUSSION TITLE
Kein Stromverbrauch mehr in Home Assistant

### DISCUSSION THREAD

**Discussion by Gesa2019** [hoylabs/OpenDTU-OnBattery#1056] [Q&A]
Hallo zusammen, 

ich hoffe, man kann mir hier weiterhelfen. 

Ich lese meinen Stromverbrauch mit dem Hitchi-Lesekopf über Tasmota aus. Per MQTT sende und sehe ich meinen Stromverbrauch in Home Assistant. Nun möchte ich nach und nach die Möglichkeiten von OpenDTU-OnBattery nutzen.

Anfangs habe ich die Stromzähler-Daten nicht in OpenDTU-OnBattery anzeigen lassen können. Nachdem ich hier im Forum ein wenig gestöbert habe, habe ich eine Lösung gefunden. Und zwar musste ich mein Tasmota-Skript leicht anpassen. Damit der Jason Pfad funktioniert, habe ich "Test" in diese Zeile eingefügt: +1,3,s,16,9600,**Test**. Siehe da, die Werte werden in OpenDTU-OnBattery angezeigt. 

Nun werden die Daten aus Tasmota aber offensichtlich nicht mehr an Home Assistant gesendet. Gibt es dafür eine Lösung? Ich hoffe, ich konnte mein Poblem einigermaßen verständlich beschreiben. Bin Laie was das Thema angeht. 

Grüße und danke für eure Unterstützung.

**Comment by spcqike** [comment]
die Daten werden weiterhin gesendet, allerdings wird HomeAssistant sie  nicht mehr zuordnen, da du den Namen geändert hast.

mit Homeassistant kenne ich mich selbst nicht aus, aber irgendwo wirst du dein ``Test`` auch dort hinterlegen müssen. Guck dir mal an, wie das Reading in HA zustande kommt. Wo es her kommt und ausgelesen wird. und passe dort den Namen entsprechend an.

**Reply by Gesa2019** [reply]
Danke für die schnelle Rückmeldung. Ich werde heute Abend mal schauen ob ich irgendwo in HA was finde, wo ich eine entsprechende Anpassung vornehmen kann.
